cmd/root.go

```diff
@@ -334,7 +335,15 @@ func initKonnectConfig() error {
 	konnectConfig.Password = password
 	konnectConfig.Debug = (viper.GetInt("verbose") >= 1)
 	konnectConfig.Address = viper.GetString("konnect-addr")
-	konnectConfig.Headers = viper.GetStringSlice("headers")
+	konnectConfig.Headers = extendHeaders(viper.GetStringSlice("headers"))
 	konnectRuntimeGroup = viper.GetString("konnect-runtime-group-name")
 	return nil
 }
+
+func extendHeaders(headers []string) []string {
+	userAgentHeader := fmt.Sprintf(
+		"User-Agent:decK/%s (%s) %s", VERSION, COMMIT, runtime.Version(),
```

Comment:
Can we skip COMMIT for now?
```suggestion
		"User-Agent:decK/%s", VERSION,
```

Reply:
done